interface/client/templates/elements/nodeInfo.html

```diff
@@ -27,7 +27,8 @@
         {{else}}
             <i class="icon-feed"></i> {{TemplateVar.get "peerCount"}} {{i18n 'mist.nodeInfo.peers'}}
             &nbsp;&nbsp;|&nbsp;&nbsp;
-            <i class="icon-layers"></i> {{formattedBlockNumber}} <i class="icon-clock" style="margin-left: 10px;"></i> {{{timeSinceBlock}}}
+            <i class="icon-layers"></i> {{formattedBlockNumber}} 
+            <br> <i class="icon-clock" style="margin-left: 10px;"></i> {{{timeSinceBlock}}}
```

Comment:
True. This was just a 'get out of the way' change, I intend to do more work on that particular piece of css in the future, specially so it resizes more elegantly when there's more content